logging/bin/deploy_opensearch.sh

```diff
@@ -328,6 +358,7 @@ if [ "$deploy_temp_masters" == "true" ]; then
        --namespace $LOG_NS \
        --values logging/opensearch/opensearch_helm_values.yaml \
        --values "$wnpValuesFile" \
+       --values "$airgapValuesFile" \
```

Comment:
Don't we need/want to reference $AIRGAP_HELM_REPO below (on line 370)?   Might be good to split line 370 below so the Helm chart name sits on its own line.